scripts/setup.sh

```diff
@@ -17,6 +17,16 @@ if [ $? != "0" ]; then
     exit 1
 fi
 
+source "$NVM_DIR/nvm.sh"
+
+if ! test -e "$NVM_DIR/nvm.sh"; then
```

Comment:
Am I right in thinking this checks for the existence of nvm.sh after already executing it?

Reply:
Oops. That line is in the wrong place 😅